src/activy/connectStrava.py

```
@@ -6,12 +9,21 @@
+    width = device.info["displayWidth"]
+    height = device.info["displayHeight"]
+    x = int(width * 12 / 13)
+    y = int(100)
+    device.click(x, y)
 
-    tryCheckStage(device, 1, templates)
-    print("Stage 1 completed")
+    try:
+        tryCheckStage(device, "mainMenu", templates)
+        raise ValueError("Didn't move to settings.")
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)